### DISCUSSION TITLE
[FR] conditional buttons

### DISCUSSION THREAD

**Discussion by Lux315** [Clooos/Bubble-Card#323] [Feature requests]
Thanks for your awesome work here.

Theres already a option to sort the horizontal button stack by motion sensor but it would be awesome if you could even set conditiona for the buttons to appear like the conditional card.

A easy "workaround" is to put the bubble card inside a conditional card, but then you can only have 1 condition with the button.

**Comment by Clooos** [comment]
Hi, you can create a template binary sensor based on multiple conditions for your conditional card. 

https://www.home-assistant.io/integrations/template/

But I'm not planning to add a feature that is very specific and already native in HA 🤓

**Comment by Lux315** [comment]
@Clooos  The idea was more about multiple buttons for multiple conditions.

Like button 1 is there if condition 1 is true

button 2 is there if condition 2 is true.

And i dont see any native way to do this because multiple horizontal buttons rows are not supported on 1 view

**Comment by Clooos** [comment]
Oh sorry indeed, I thought that you asked that for the buttons and not the horizontal buttons stack. It could be a nice new feature indeed!

**Comment by Lux315** [comment]
Ahh im sorry for the confusion!

Let me know if / when you implement it and ill be there for testing!

Thx!

**Reply by zbuh** [reply]
How about conditional sub-buttons, this would allow a great card for showing notifications like... :D

**Comment by Lux315** [comment]
@Clooos Do you plan to implement something like this soon?

**Reply by Clooos** [reply]
I don't know, but it's maybe possible to achieve this with a conditional custom template.

**Reply by Lux315** [reply]
Hi @Clooos 

I tried to do this with a template but i cant get it to work.

How can i adress a single button in the HBS to give it a template based on a state?

**Comment by Impudicus** [comment]
Anything new on this topic?

**Comment by tbowmo** [comment]
I could also use conditional visibility of individual buttons in horizontal button stack. Right now, I do it with css, setting display: none on the button I want to be hidden. But that is suboptimial, as it makes a mess in the rendering
